ra-optimisation/rao-api/src/main/java/com/farao_community/farao/rao_api/parameters/MaxMinRelativeMarginParameters.java

```diff
@@ -11,20 +11,20 @@
  * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
  */
 public class MaxMinRelativeMarginParameters extends MaxMinMarginParameters {
-    private final double negativeMarginObjectiveCoefficient;
     private final double ptdfSumLowerBound;
+    private final double highestThreshold;
```

Comment:
you only need this inside MaxMinRelativeMarginFiller. You can compute it there

Reply:
getLargestCnecThreshold is defined in Castor (used for UnoptimizedCnecParam), and calling Castor in Parameters generates a circular reference. And UnoptimizedCnecParam + MaxMinRelativeMarginParam don't have a common parent